README.md

```diff
@@ -63,25 +67,66 @@ This contract has to be initialized with the following parameters:
 
 * `oct_token_contract`: The account id of OCT token contract.
 
-The `oct_token_contract` should be stored in this contract for using in [Callback function 'ft_on_transfer'](#callback-function-ft_on_transfer).
+The `oct_token_contract` should be stored in this contract for using in [Callback function of token transfer](#callback-function-of-token-transfer).
 
-### Register an appchain
+### Change value of minimum register deposit
+
+This action needs the following parameters:
+
+* `value`: The value of `minimum register deposit`.
+
+Qualification of this action:
+
+* The `sender` must be the `owner`.
+
+The `minimum register deposit` is set to `value`.
+
+### Update metadata of an appchain
 
 This action needs the following parameters:
 
 * `appchain_id`: The unique identity in Octopus Network. It cannot be duplicated with any other registered appchain.
 * `website_url`: The url of the official website of the appchain.
-* `github_address`: The address of the github repository of the appchain, if it is an open-source project.
-* `github_release`: The release vesion of the github repository of the appchain, if it is an open-source project.
+* `github_address`: The address of the github repository of the appchain.
+* `github_release`: The release vesion of the github repository of the appchain.
 * `commit_id`: The commit id of source code of the github repository of the appchain, if it is an open-source project.
 * `contact_email`: The email of the contact of the appchain project, which is used for communidating with the appchain team.
 
-If the parameters are all valid, the appchain will be registered to this contract. These data will be saved to the metadata of the new appchain.
+Qualification of this action:
+
+* The `sender` must be current `appchain owner` of `appchain basedata` corresponding to `appchain_id`.
+* The `appchain state` of `appchain basedata` corresponding to `appchain_id` must be `registered`.
+
+The metadata will be updated to `appchain basedata` corresponding to `appchain_id`.
+
+Generate log: `The metadata of appchain <appchain_id> is updated.`
+
+### Approve an appchain to start auditing
+
+This action needs the following parameters:
+
+* `appchain_id`: The id of an appchain.
+
+Qualification of this action:
+
+* The `sender` must be the `owner`.
+* The `appchain state` of `appchain basedata` corresponding to `appchain_id` must be `registered`.
+
+The `appchain state` of `appchain basedata` corresponding to `appchain_id` is set to `auditing`.
+
+Generate log: `Appchain <appchain_id> starts auditing.`
+
+### Register an appchain
+
+The `appchain owner` can transfer a certain amount (not less than `minimum register deposit`) of OCT token to this contract by calling function `ft_transfer_call` of `oct_token_contract`. And the calling param `msg` MUST be `register appchain <appchain_id>,<website_url>,<github_address>,<github_release>,<commit_id>,<contact_email>`.
```

Comment:
Will anyone deposit more than the minimum amount?

Reply:
Change it to 'must be equal to'.